OSDOCS 11336: Updated Operator catalogs for oc-mirror (#88265)
openshift/openshift-docs: modules/oc-mirror-operator-catalog-filtering.adoc

     packages: 
    - name: compliance-operator
----

> |One bundle, corresponding to the head version of the default channel for that package.

Reviewer: ```suggestion
|One bundle, corresponding to the head version for each channel of that package.
```

        channels
          - name: stable
----

> |The head bundle for the selected channel of that package. The `defaultChannel` should be used in case the filtered channels are not the default.

Reviewer 2: ```suggestion
|The head bundle for the selected channel of that package. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

- Avoid using should
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

- Object name must be specified per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references
- Use active voice

            channels:
               - name: 'stable-v0'
----
|All bundles for the packages and channels specified.

> The `defaultChannel` should be used in case the filtered channels are not the default.

Reviewer 2: ```suggestion
You must use the `defaultChannel` field in case the filtered channels are not the default.
```

- Avoid using should
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed. For example, when parameters must be reset, write “Reset the parameters” and not “You should reset the parameters”.

- Object name must be specified per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references
- Use active voice

          - name: stable
            minVersion: 5.6.0
----

> |Within the selected channel of that package, all versions starting with the `minVersion` up to the channel head. The `defaultChannel` should be used in case the filtered channels are not the default.

Reviewer 2: ```suggestion
|Within the selected channel of that package, all versions starting with the `minVersion` up to the channel head. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

Same comment

          - name: stable
            maxVersion: 6.0.0
----

> |within the selected channel of that package, all versions up to `maxVersion`:

Reviewer 3: ```suggestion
|Within the selected channel of that package, all versions up to `maxVersion`.
```

Head of channel is not included, even if multiple channels are included in the filtering.
You should expect errors if this filtering leads to a channel with multiple heads.

> 

Reviewer: The `defaultChannel` should be used in case the filtered channels are not the default.

> You should expect errors if this filtering leads to a channel with multiple heads. The `defaultChannel` should be used in case the filtered channels are not the default.

Reviewer 2: ```suggestion
You might see errors if this filtering leads to a channel with multiple heads. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

Same comment

            minVersion: 5.6.0
            maxVersion: 6.0.0
----
|Within the selected channel of that package, all versions between the `minVersion` and `maxVersion`. The head of channel is not included, even if multiple channels are included in the filtering.

> You should expect errors if this filtering leads to a channel with multiple heads.

Reviewer: The `defaultChannel` should be used in case the filtered channels are not the default.

> You should expect errors if this filtering leads to a channel with multiple heads. The `defaultChannel` should be used in case the filtered channels are not the default.

Reviewer 2: ```suggestion
You might see errors if this filtering leads to a channel with multiple heads. You must use the `defaultChannel` field in case the filtered channels are not the default.
```

Same coment
